internal/podman/client.go

```diff
@@ -29,8 +29,10 @@ import (
 )
 
 var (
-	ErrPodmanNotFound  = errors.New("podman not found in your system, check requirements at http://docpage")
-	ErrNetworkNotFound = errors.New("network ip range was not found")
+	ErrPodmanNotFound    = errors.New("podman not found in your system, check requirements at http://docpage")
+	ErrNetworkNotFound   = errors.New("network ip range was not found")
+	defaultMachineCPUs   = "2"
+	defaultMachineMemory = "2048"
```

Comment:
how do you know if this machine has enough memory or CPU cores?

Reply:
oh good point, I will add a check. Thanks

Reply:
updated

Reply:
this actually need a bit more love, I will update you as soon as it is ready for review. Sorry for the ping

Reply:
updated